utils/wasm-optimizer/src/cargo_toolchain.rs

```diff
@@ -94,16 +94,11 @@ impl Toolchain {
             self.raw_toolchain_str() == toolchain,
             anyhow!(
                 "recommended toolchain `{x}` not found, install it using the command:\n\
-        rustup toolchain install {x} --component llvm-tools --target wasm32-unknown-unknown\n\n\
+        rustup toolchain install {x} --component llvm-tools --target wasm32v1-none\n\n\
```

Comment:
there is no point in advising to install the llvm-tools component, which is only used for runtime fuzzer to ecosystem developer